files/default/setup-ephemeral-drives.sh

```diff
@@ -21,6 +21,12 @@ function error_exit () {
   exit 1
 }
 
+function exec_command() {
+  _command_output=$($@ 2>&1)
+  _exit_code=$?
+
+  [[ $_exit_code -ne 0 && $(echo "${_command_output}" | grep -i "you should reboot now") ]] && RC=1
```

Comment:
can we add a small comment on why we are doing this? especially why excluding this specific error? Probably you mentioned it in the PR but having it in the code might be useful for future reference.